code/datums/abilities/flock/flockmind.dm

```diff
@@ -299,13 +299,17 @@
 
 /datum/targetable/flockmindAbility/controlPanel/cast(atom/target)
 	if(..())
-		return 1
-	var/client/user = holder.owner.client
+		return TRUE
+	// var/client/user = holder.owner.client
 	var/mob/living/intangible/flock/flockmind/F = holder.owner
-	var/chui/window/flockpanel/panel = F.flock.panel
-	if(isnull(user) || isnull(F) || isnull(panel))
-		return 1
-	panel.Subscribe(user)
+	F.flock.flockpanel = tgui_process.try_update_ui(F, F, F.flock.flockpanel)
+	if (!F.flock.flockpanel)
+		F.flock.flockpanel = new(F, F, "FlockPanel")
+		F.flock.flockpanel.open()
+	// var/chui/window/flockpanel/panel = F.flock.panel
+	// if(isnull(user) || isnull(F) || isnull(panel))
+		// return 1
+	// panel.Subscribe(user)
```

Comment:
```suggestion
```
Same with all of the other commented out CHUI, just remove them, don't need the comments since we have the blame/diff.

Reply:
good point, I'll strip out all the CHUI code